### PR TITLE
test: Update to reworked pixel test machinery

### PR DESCRIPTION
**Pull request by mvollmer**
TEST_MOBILE is not a thing anymore, but assert_pixels switches to the
"mobile" layout temporarily.

### CODE REVIEW

**Comment by mvollmer** [comment]
Draft because we might catch up with more features in one go, like the "medium" layout.

**Comment by mvollmer** [comment]
> Draft because we might catch up with more features in one go, like the "medium" layout.

Let's not, that will take a bit longer I guess.

File: test/check-machines-create [hunk 1]
```diff
@@ -931,6 +931,8 @@ class TestMachinesCreate(VirtualMachinesCase):
             if self.sourceTypeSecondChoice:
                 b.select_from_dropdown("#source-type", self.sourceTypeSecondChoice)
 
+            # Data loading doesn't start immediately
+            time.sleep(1)
             b.wait_attr("#os-select-group", "data-loading", "false")
             # For the mocked fedora URL installation wait to see that the OS section is automatically filled
```
Comment (skobyda) [inline test/check-machines-create:937]: Just a question, but with this sleep, are we waiting for OS data to be loaded? I thought that's what the following              `b.wait_attr("#os-select-group", "data-loading", "false")` does? Or maybe it's broken?

Comment (mvollmer) [reply test/check-machines-create:937]: This sleep waits for the loading to _start_.  The following `wait_attr` then waits for the loading to be finished.

If you look at the current pixel test reference in main, you can see that the test catches the dialog with the OS field disabled, which means that the data is currently being loaded, no?

image

Comment (skobyda) [reply test/check-machines-create:937]: Ah right I see. It seems it's a result of how `onOsAutodetect` sets `autodetectOSInProgress` value. Perhaps that could be somehow rewritten so there is no sleep, but that's outside of this PR.

**Review by skobyda — APPROVED** [review]

File: test/check-machines-create (hunk 1 above)
Comment (skobyda) [reply test/check-machines-create:937]: Ah right I see. It seems it's a result of how onOsAutodetect sets autodetectOSInProgress value. Perhaps that could be somehow rewritten so there is no sleep, but that's outside of this PR.

**Review by martinpitt — APPROVED** [review]
Many thanks! packit is busted right now, landing.

**Pull request merged**